core/src/main/scala/com/softwaremill/react/kafka/ReactiveKafkaConsumer.scala

```diff
@@ -15,7 +32,18 @@ case class ReactiveKafkaConsumer[K, V](properties: ConsumerProperties[K, V]) {
       properties.keyDeserializer,
       properties.valueDeserializer
     )
-    c.subscribe(List(properties.topic)) // support multiple topics?
+
+    if (topicPartitionOffset.isInstanceOf[TopicPartitionOffset]) {
+      c.seek(new TopicPartition(topicPartitionOffset.topic, topicPartitionOffset.partition), topicPartitionOffset.offset)
+    }
+    else if (Option(topicsAndPartitions).isDefined && topicsAndPartitions.length > 0) {
+      val topicsPartitions = topicsAndPartitions.map(tp => tp.toTopicPartition).toList
+      c.assign(topicsPartitions)
+    }
+    else {
+      val topics = properties.topic.split(",")
```

Comment:
You're saying `ConsumerProperties.topic` to `ConsumerProperties.topics` - just to make sure I understand correctly.